src/get-started/java-binding.md

```diff
@@ -145,8 +145,6 @@ rules – see the corresponding section of our [documentation][transactions].
 
 Transactions are transmitted by external service clients to the framework as
 Exonum messages.
-<!-- TODO: Link a section on messages in the main Exonum docs
```

Comment:
Why do you remove this?

Reply:
Thanks, should've put a link here.
@Maria-Nosyk what would be a correct link to a messages section in 0.10 documentation? Would it stay the same - https://exonum.com/doc/version/0.10/architecture/consensus/#messages ?